swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
+CONDITION_DESCRIPTION = {0:"One person's trash is another's treasure.",
+1:"Probably not it's first swap meet.",
+2:"Stained but functional",
+3:"Used enough to know it works juuust right.",
+4:"Stored for ages and finally seeing the light.",
+5:"The tag is still on it."}
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, category = "", condition = 0):
+            self.category = category
+            self.condition = condition
+
+
+    def __str__(self):
+        return "Hello World!"
+
+
+    def condition_description(self):
+        for key, value in CONDITION_DESCRIPTION.items():
+            if self.condition is key:
+                return value
```

Comment:
Since `CONDITION_DESCRIPTION` is a dictionary, we could simplify this to avoid looping over the values: 
```suggestion
        return CONDITION_DESCRIPTION[self.condition]
```
Since we don't currently check if the condition passed to an Item is strictly between 0-5, we might want to wrap it in a try/except in case we get a KeyError.

Something else to consider: what would happen if we tried to get the description for an Item with a condition of 3.5? There are tests that set the condition to 3.5, but don't try to read the description. The description test checks for a condition of 1 and a description of 5, but we should still make sure our code covers any other reasonable conditions, including floating point values (since we've seen them used in other tests).

How could we modify this (slightly) to cover the values between the whole numbers?